test/Microsoft.AspNet.Mvc.ViewFeatures.Test/Rendering/HtmlHelperRadioButtonExtensionsTest.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.Mvc.Rendering;
+using Microsoft.AspNet.Mvc.TestCommon;
+using Xunit;
+
+namespace Microsoft.AspNet.Mvc.Core
+{
+    /// <summary>
+    /// Test the RadioButton extensions in <see cref="HtmlHelperInputExtensions" /> class.
+    /// </summary>
+    public class HtmlHelperRadioButtonExtensionsTest
+    {
+        [Fact]
+        public void RadioButtonHelpers_UsesSpecifiedExpression()
```

Comment:
This test doesn't demonstrate what you've got on the tin. Here, you're showing that `checked` isn't generated when `Property1` is something (anything) other than `"myvalue"`. Also need a test showing that `checked` _is_ generated when `Property1 == "myvalue"`. And it's also worth confirming whether the conditions are case-sensitive i.e. test what happens when `Property1 == "MyValue"`.